app/src/main/java/me/hufman/androidautoidrive/music/controllers/SpotifyAppController.kt

```diff
@@ -422,7 +434,7 @@ class SpotifyAppController(context: Context, val remote: SpotifyAppRemote, val w
 			MusicAction.SKIP_TO_QUEUE_ITEM -> false
 			MusicAction.SET_SHUFFLE_MODE -> playerActions?.canToggleShuffle == true
 			MusicAction.SET_REPEAT_MODE -> playerActions?.canRepeatContext == true || playerActions?.canRepeatTrack == true
-			// figure out search
+			MusicAction.PLAY_FROM_SEARCH -> true
```

Comment:
The SpotifyAppController shouldn't declare support for this because its `playFromSearch` function doesn't do anything. This menu entry is shown because the GenericAppController controlling Spotify's MediaSession provides the functionality for `playFromSearch`.

Reply:
Ah good point, removed this MusicAction